TestProjects/com.unity.mobile-notifications-sample/Scripts/iOSTest.cs

```diff
@@ -338,22 +338,18 @@ IEnumerator GetCurrentLocation()
         }
 
         //Request authorization if it is not enabled in Editor UI
-        IEnumerator RequestAuthorization(AuthorizationOption options =
-                AuthorizationOption.Alert | AuthorizationOption.Badge | AuthorizationOption.Sound)
+        IEnumerator RequestAuthorization(AuthorizationOption options)
         {
             m_LOGGER.Blue($"[{DateTime.Now.ToString("HH:mm:ss.ffffff")}] Call {MethodBase.GetCurrentMethod().Name}");
-            AuthorizationRequest request = new AuthorizationRequest(options, true);
-            yield return request;
-            using (AuthorizationRequest req =
-                       new AuthorizationRequest(AuthorizationOption.Alert | AuthorizationOption.Badge, true))
+            using (AuthorizationRequest request = new AuthorizationRequest(options, true))
             {
-                while (!req.IsFinished) { yield return null; };
-                if (req.Granted)
+                while (!request.IsFinished) { yield return null; };
```

Comment:
since you're touching the code, split this in multiple lines, will make it easier to place breakpoints

also ; is not needed at the end here

Reply:
Fixed.